scalding-core/src/test/scala/com/twitter/scalding/mathematics/Matrix2Test.scala

```diff
@@ -40,6 +44,37 @@ class Matrix2Sum(args: Args) extends Job(args) {
   sum.write(TypedText.tsv[(Int, Int, Double)]("sum"))
 }
 
+class Matrix2SumOrderedSerialization(args: Args) extends Job(args) {
+
+  import Matrix2._
+  import cascading.pipe.Pipe
+  import cascading.tuple.Fields
+  import com.twitter.scalding.TDsl._
+
+  override def config = super.config + (Config.ScaldingRequireOrderedSerialization -> "true")
+  implicit val intOS: OrderedSerialization[Int] = new OrderedSerialization[Int] {
+    def read(in: InputStream) = Try(Reader.read[Int](in))
+    def write(o: OutputStream, t: Int) = Try(Writer.write[Int](o, t))
+    def hash(t: Int) = t.hashCode
+    def compare(a: Int, b: Int) = java.lang.Integer.compare(a, b)
+    def compareBinary(a: InputStream, b: InputStream) =
+      OrderedSerialization.readThenCompare(a, b)(this)
+    val staticSize = Some(4)
+    def dynamicSize(i: Int) = staticSize
+  }
+
+  val p1: Pipe = Tsv("mat1", ('x1, 'y1, 'v1)).read
```

Comment:
this style is not recommended any more (.toTypedPipe), so let's just use TypedText.tsv the whole way:
```scala
val tp1 = TypedPipe.from(TypedText.tsv[(Int, Int, Double)]("mat1"))
```